packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.tsx

```diff
@@ -1,65 +1,183 @@
-                checked: !this.state.checked,
-            });
-        }
-    }
-
-    render() {
-        return (
-            <div>
-                <div className="form-group">
-                    <label className="form-control-label">Checkbox set</label>
-                    <br />
-                    <Checkbox>
-                        <Label classes={['label']}>An unchecked checkbox</Label>
-                    </Checkbox>
-                    <br />
-                    <Checkbox classes={['mt1']} checked={true}>
-                        <Label classes={['label']}>A checked checkbox</Label>
-                    </Checkbox>
-                    <br />
-                    <Checkbox classes={['mt1']} disabled={true}>
-                        <Label classes={['label']}>A disabled checkbox</Label>
-                    </Checkbox>
-                    <br />
-                    <Checkbox classes={['mt1']} checked={true} disabled={true}>
-                        <Label classes={['label']}>A checked and disabled checkbox</Label>
-                    </Checkbox>
-                    <br />
-                    <Checkbox classes={['mt1']} indeterminate={true}>
-                        <Label classes={['label']}>A checked and indeterminate (partially selected) checkbox</Label>
-                    </Checkbox>
-                    <br />
-                    <Checkbox classes={['mt1']} checked={this.state.checked} onClick={() => this.handleClick()}>
-                        <Label classes={['label']}>A checkbox with state</Label>
-                    </Checkbox>
-                </div>
-                <div className="form-group">
-                    <label className="form-control-label">A checkbox with no label</label>
-                    <br />
-                    <Checkbox />
-                </div>
-            </div>
-        );
-    }
-}
+import * as React from 'react';
+
+import {ReactVaporStore} from '../../../../docs/ReactVaporStore';
+import {Button} from '../../button/Button';
+import {Label} from '../../input/Label';
+import {LabeledInput} from '../../input/LabeledInput';
+import {Section} from '../../section/Section';
+import {Checkbox} from '../Checkbox';
+import {CheckboxConnected} from '../CheckboxConnected';
+import {toggleDisabledAllGroupedCheckbox} from '../GroupableCheckboxActions';
+import {GroupableCheckboxConnected} from '../GroupableCheckboxConnected';
+
+export interface ICheckboxWithState {
+    checked?: boolean;
+}
+
+export class CheckboxExamples extends React.Component {
+    static description = 'Checkboxes allow users to select multiple options from a set.';
+
+    render() {
+        return (
+            <Section title="Checkbox set">
+                <Checkboxset />
+                <CheckboxConnectedExamples />
+                <GroupableCheckboxConnectedExamples />
+            </Section>
+        );
+    }
+}
+
+const Checkboxset: React.FunctionComponent<ICheckboxWithState> = () => {
```

Comment:
The first generic is the type of the `Props`, but here you don't have any props

Reply:
I'm wondering if we should discard the non connected examples 🤔 Or at least those that don't have any behaviour

Reply:
:+1: